environment_setup/install_openocd.sh

```diff
@@ -21,6 +21,8 @@ git clone https://github.com/ntfreak/openocd.git $OPENOCD_TMP_DIR
 
 # Build and Install Openocd
 cd $OPENOCD_TMP_DIR
+sudo cp contrib/60-openocd.rules /etc/udev/rules.d/
+sudo adduser $USER plugdev
```

Comment:
Could we move this into a section above called `# Set permissions to allow this user to access USB devices`?

Reply:
done